From e93c0e0a1800c50ec2362d871909fdcf6192e82d Mon Sep 17 00:00:00 2001
From: Carlos Santos <casantos@datacom.ind.br>
Date: Thu, 13 Apr 2017 20:19:37 -0300
Subject: [PATCH] scripts/config.sub: recognize three additional ARM cores

ARMv6-M (armv6m), ARMv7E-M (armv7em), ARMv8-A (armv8a), so for a bare-metal
toolchain we get the following result:

    armv7em-eabi -> armv7em-unknown-eabi

Test with the following defconfig:

    CT_ARCH_arm=y
    CT_ARCH_SUFFIX="v7em"
    CT_LIBC_none=y

Signed-off-by: Carlos Santos <casantos@datacom.ind.br>
---
 scripts/config.sub | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/config.sub b/scripts/config.sub
index 87abeab..d868d04 100755
--- a/scripts/config.sub
+++ b/scripts/config.sub
@@ -253,6 +253,7 @@ case $basic_machine in
 	| am33_2.0 \
 	| arc | arceb \
 	| arm | arm[bl]e | arme[lb] | armv[2-8] | armv[3-8][lb] | armv7[arm] \
+	| armv6m | armv7em | armv8a \
 	| avr | avr32 \
 	| ba \
 	| be32 | be64 \
-- 
2.7.4

